ethereum/core/src/main/java/org/hyperledger/besu/ethereum/transaction/TransactionSimulator.java

```diff
@@ -234,7 +234,7 @@ public TransactionSimulator(
     // fail.
     if (GoQuorumOptions.goQuorumCompatibilityMode && value.isZero()) {
       Gas privateGasEstimateAndState =
-          protocolSpec.getGasCalculator().getMaximumTransactionCost(32);
+          protocolSpec.getGasCalculator().getMaximumTransactionCost(64);
```

Comment:
Is this expected?

Reply:
I had hard coded the wrong number - https://github.com/hyperledger/besu/blob/fd59570e2cb30425bcae5f231e4d22e9ef037410/ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/IstanbulGasCalculator.java#L145